Examples/LegacySwiftExample

```diff
@@ -1 +1 @@
-../IntegrationTests/PurchaseTester
\ No newline at end of file
+../TestingApps/PurchaseTester/
```

Comment:
There's no trailing `/` in the old version, not sure if it matters 🤔

Reply:
It seems to work fine anyway, but I’ll update just in case